### PR TITLE
Set up

### PR DESCRIPTION
**Pull request by clin2on3mun**
in this milestone:

- Choose REST Countries API
- followed the given Design and add some other features
- created a Home page and displayed items
- created a detail page of displayed items
- did testing of components and redux

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            key={country.cca3}
+            onClick={() => navigate(`/country/${country.cca3}`)}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter') {
+                navigate(`/country/${country.cca3}`);
+              }
+            }}
+            tabIndex={0}
+            role="button"
+          >
```
Comment (thecodechaser) [inline src/components/Home.js:46]: - [ ] The search functionality is not working as expected, I'm entering the search text as countries name and it's not giving desired results. Please make sure to fix this issue.

image

image

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          <div
+            className="country-card"
+            key={country.cca3}
+            onClick={() => navigate(`/country/${country.cca3}`)}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter') {
+                navigate(`/country/${country.cca3}`);
+              }
+            }}
+            tabIndex={0}
```
Comment (thecodechaser) [inline src/components/Home.js:44]: - [ ] Please make sure to add a mic, settings, etc. icons on the header to follow the design guidelines. These should not be functional but only for design purposes.

### Your design

image

### Given design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @clin2on3mun👋, 

### You are doing a marvelous job on the React capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image


## To Highlight 🎉
- All tests are passed. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Highly organized code. ✔️
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] I would suggest adding steps in the README file to set up and run the app, It will make the README file more professional. Example: `npm install` `npm run start`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @clin2on3mun, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**